bookwyrm/settings.py

```diff
@@ -174,8 +184,10 @@
 
 PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
 STATIC_URL = "/static/"
+STATIC_PATH = "%s/%s" % (DOMAIN, env("STATIC_ROOT", "static"))
 STATIC_ROOT = os.path.join(BASE_DIR, env("STATIC_ROOT", "static"))
 MEDIA_URL = "/images/"
+MEDIA_PATH = "%s/%s" % (DOMAIN, env("MEDIA_ROOT", "images"))
 MEDIA_ROOT = os.path.join(BASE_DIR, env("MEDIA_ROOT", "images"))
```

Comment:
It might be interesting to think a bit more on how we access the assets and images, it will be useful when there's other types of hosting, but it's out of scope for this PR

Reply:
agreed!